saleor/product/tasks.py

```diff
@@ -68,9 +68,3 @@ def update_products_minimal_variant_prices_of_discount_task(discount_pk: int):
 def update_products_minimal_variant_prices_task(product_ids: List[int]):
     products = Product.objects.filter(pk__in=product_ids)
     update_products_minimal_variant_prices(products)
-
-
-@app.task
-def update_all_products_minimal_variant_prices_task():
```

Comment:
AFAIK this task was created to future usage from celery beat. But I am not sure. You can ask @derenio

Reply:
This is true, although I'm not sure what's the status of shipping saleor with something like `celery beat` for which this task was designed. I think we can remove it. When we implement the default `periodic tasks` schema we will have to think about the _minimal variant price_ recalculations.